frontend/src/container/LogDetailedView/utils.tsx

```diff
@@ -251,18 +251,30 @@ export const getDataTypes = (value: unknown): DataTypes => {
 };
 
 export const removeEscapeCharacters = (str: string): string =>
-	str.replace(/\\([ntfr'"\\])/g, (_: string, char: string) => {
-		const escapeMap: Record<string, string> = {
-			n: '\n',
-			t: '\t',
-			f: '\f',
-			r: '\r',
-			"'": "'",
-			'"': '"',
-			'\\': '\\',
-		};
-		return escapeMap[char as keyof typeof escapeMap];
-	});
+	str
+		.replace(/\\x1[bB][[0-9;]*m/g, '')
+		.replace(/\\u001[bB][[0-9;]*m/g, '')
+		.replace(/\\x[0-9A-Fa-f]{2}/g, '')
+		.replace(/\\u[0-9A-Fa-f]{4}/g, '')
+		.replace(/\\[btnfrv0'"\\]/g, '');
+
+export const unescapeString = (str: string): string =>
```

Comment:
Add code documentation

Reply:
inline comments added below.. will add more reasoning behind this

Reply:
Yes , let's try adding documentation into the code for better readability.

Reply:
done ✅